recipe/meta.yaml

```diff
@@ -13,33 +7,35 @@ package:
   version: {{ cuda_compiler_version }}
 
 build:
-  number: 25
+  number: 26
   skip: {{ not (compiler("cuda").startswith("nvcc") and cuda_compiler_version == "11.8") }}
 
 outputs:
-  - name: nvcc_{{ cross_target_platform }}
-    script: linux/install_nvcc.sh     # [linux]
-    script: windows/install_nvcc.bat  # [win]
+  - name: {{ cuda_compiler }}_{{ cross_target_platform }}
+    {% set script = "linux/install_nvcc.sh" %}     # [linux]
+    {% set script = "windows/install_nvcc.bat" %}  # [win]
+    script: {{ script }}
```

Comment:
This fixes a linter hint with souschef above: https://github.com/conda-forge/nvcc-feedstock/pull/115#issuecomment-2608390342